packages/workbox-webpack-plugin/src/inject-manifest.js

```diff
@@ -37,12 +37,81 @@ const _generatedAssetNames = new Set();
  * @module workbox-webpack-plugin
  */
 class InjectManifest {
+  // eslint-disable-next-line jsdoc/newline-after-description
   /**
-   * Creates an instance of InjectManifest.
+   * Creates an instance of GenerateSW.
    *
-   * @param {Object} [config] See the
-   * [configuration guide](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin#configuration)
-   * for all supported options and defaults.
+   * @param {Object} config The configuration to use.
+   *
+   * @param {string} config.swSrc An existing service worker file that will be
+   * compiled and have a precache manifest injected into it.
+   *
+   * @param {Array<workbox-build~ManifestEntry>} [config.additionalManifestEntries]
```

Comment:
Same comments as above w.r.t `module:workbox-build` (here and throughout this file).